src/main/java/dinkplugin/notifiers/LeaguesNotifier.java

```diff
@@ -0,0 +1,290 @@
+                : "%USERNAME% completed a %TIER% task: %TASK%.")
+            .replacementBoundary("%")
+            .replacement("%USERNAME%", Replacements.ofText(playerName))
+            .replacement("%TIER%", Replacements.ofText(tier.getDisplayName()))
+            .replacement("%TASK%", Replacements.ofWiki(task, "Trailblazer_Reloaded_League/Tasks"))
+            .replacement("%TROPHY%", newTrophy
+                ? Replacements.ofWiki(trophy.getValue(), String.format("Trailblazer reloaded %s trophy", trophy.getValue().toLowerCase()))
+                : Replacements.ofText("?"))
+            .build();
+        createMessage(config.leaguesSendImage(), NotificationBody.builder()
+            .type(NotificationType.LEAGUES_TASK)
+            .text(text)
+            .extra(new LeaguesTaskNotificationData(task, tier, taskPoints, totalPoints, tasksCompleted, tasksUntilNextArea, pointsUntilNextRelic, pointsUntilNextTrophy, justEarnedTrophy))
+            .playerName(playerName)
+            .seasonalWorld(true)
+            .build());
+    }
+
+    /**
+     * @return the number of areas that have been unlocked as integer and human name
+     */
+    private Map.Entry<Integer, String> numAreasUnlocked() {
+        // While Jagex's code has 5 areas (2 default, 3 discretionary),
+        // most players think just in terms of the 3 discretionary areas,
+        // so we disregard Misthalin and consider Karamja as the zeroth area.
+        // Thus, the number of unlocked areas is bounded by 3 (instead of 5).
+        if (client.getVarbitValue(FIVE_AREAS) > 0) {
+            return Map.entry(3, ith(3));
+        }
+        if (client.getVarbitValue(FOUR_AREAS) > 0) {
+            return Map.entry(2, ith(2));
+        }
+        if (client.getVarbitValue(THREE_AREAS) > 0) {
+            return Map.entry(1, ith(1));
+        }
+        if (client.getVarbitValue(TWO_AREAS) > 0) {
+            return Map.entry(0, ith(0)); // Karamja
+        }
+        return null;
+    }
+
+    private static String ith(int i) {
+        if (i == 0) return "zeroth";
+        if (i == 1) return "first";
+        if (i == 2) return "second";
+        if (i == 3) return "third";
+        if (i == 4) return "fourth";
+        if (i == 5) return "fifth";
+        return String.valueOf(i);
+    }
+
+    static {
+        AREA_BY_TASKS = Collections.unmodifiableNavigableMap(
+            new TreeMap<>(Map.of(0, 0, 60, 1, 200, 2, 400, 3))
+        );
+
+        NavigableMap<Integer, String> thresholds = new TreeMap<>();
+        thresholds.put(2_500, "Bronze");
+        thresholds.put(5_000, "Iron");
+        thresholds.put(10_000, "Steel");
+        thresholds.put(18_000, "Mithril");
+        thresholds.put(28_000, "Adamant");
+        thresholds.put(42_000, "Rune");
+        thresholds.put(56_000, "Dragon");
+        TROPHY_BY_POINTS = Collections.unmodifiableNavigableMap(thresholds);
+
+        TIER_BY_RELIC = Map.ofEntries(
```

Comment:
In a future PR, can switch hardcoded relic names to dynamic lookup from structs/enums